components/SignInForm.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import shortid from 'shortid';
+
+const renderField = ({ input, label, type, meta: { touched, error } }) =>
+  <div>
+    <input {...input} placeholder={label} type={type} />
+    {touched &&
+      error &&
+      <div>
+        {error}
+      </div>}
+  </div>;
+
+const renderErrors = errors =>
+  <div className="alert alert-danger" role="alert">
+    {errors &&
+      errors.map(err =>
+        <span key={shortid.generate()}>
+          {err.message}
+        </span>
+      )}
+  </div>;
+
+let AuthForm = props => {
```

Comment:
It could be good if it's been different component so we can use it where we want to show sign in.